### PR TITLE
Add examples for stopping sound and creating books

### PR DESCRIPTION
**Pull request by KingOfSquares**
Not native in English, could probably use a proofreading.

### CODE REVIEW

File: source/book.rst [hunk 1]
```diff
@@ -1,3 +1,26 @@
 =====
 Books
-=====
\ No newline at end of file
+=====
+
+Books in adventure is not necessarily connected to an interactable book item in the client.
```
Comment (MiniDigger) [inline source/book.rst:5]:
```suggestion
Books in adventure are not necessarily connected to an interactable book item in the client.
```

File: source/book.rst [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+As of the current release such a connection needs to be implemented outside of adventure.
+
+
+
+Creating a Book
+---------------
+
+Any component that surpass the game limit for text per page will be truncated client side, the same applies
```
Comment (MiniDigger) [inline source/book.rst:13]:
```suggestion
Any component that surpasses the game limit for text per page will be truncated client side, the same applies
```

File: source/book.rst [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+to the amount of components(pages). Further reading about these limits can be done at the `minecraft wiki <https://minecraft.gamepedia.com/Book_and_Quill#Writing>`_
+
+.. code:: java
+
+    //Create and open a book about cats for the target audience
+    public void openMyBook(final @NonNull Audience target){
+        Component bookTitle = TextComponent.text("Encyclopedia of cats");
+        Component bookAuthor = TextComponent.text("catlovr2000");
```
Comment (MiniDigger) [inline source/book.rst:21]:
```suggestion
        Component bookAuthor = TextComponent.text("catlover2000");
```
not sure if on purpose, lol

Comment (kashike) [reply source/book.rst:21]: Let's go with a name we know instead - `kashike` works.

**Review by kashike — APPROVED** [review]

**Comment by kashike** [comment]
Thanks!

**Pull request merged**